### PR TITLE
Scope SCSS in Create.vue and adjust Results.vue

### LINKED ISSUE #1442: Scope CSS in Create.vue
CSS in Create.vue is not scoped and thus spitting anywhere. Most probably needs CSS adaptions in other places to correct for visual changes.

### PR DESCRIPTION
**Pull request by Chartman123**
Fixes #1442 

Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

File: src/views/Results.vue [hunk 1]
```diff
@@ -327,20 +327,39 @@ export default {
...
+		h2 {
+			margin-bottom: 0; // because the input field has enough padding
+			font-size: 28px;
+			font-weight: bold;
+			margin-top: 32px;
+			padding-left: 14px;
+			padding-bottom: 8px;
+			overflow: hidden;
+			text-overflow: ellipsis;
+			white-space: nowrap;
```
Comment (jotoeri) [inline src/views/Results.vue:357]: Not necessary? 🤔

**Review by jotoeri — APPROVED** [review]
Thanks for tackling directly! :)
- One small thing commented, but that was already like that before as it seems. 🤷‍♂️

File: src/views/Results.vue (hunk 1 above)
Comment (Chartman123) [reply src/views/Results.vue:357]: With those rules:
grafik
Without:
grafik

Comment (jotoeri) [reply src/views/Results.vue:357]: Ah, sorry - I just clicked and meant the `white-space` line.
Regarding the title, i think i'd leave it here as is. There's no need to show the full title here again, no?

Comment (Chartman123) [reply src/views/Results.vue:357]: For me the shortened version is fine. :)

**Review by susnux — APPROVED** [review]

**Pull request merged**